gomock_generator/gomockgenerator/generate.go

```diff
@@ -79,7 +86,10 @@ func GenerateMocks(ctx context.Context, gcfg GenerationConfiguration, mocksCfg M
 	}).Infof("Generating %v mocks", len(mocksCfg.Mocks))
 
 	var mockSigs map[string]string
-	mockSigsPath := path.Join(os.Getenv("GOPATH"), "src", mocksCfg.BaseDirectory, gcfg.SignaturesFilename)
+	mockSigsPath := path.Join(gcfg.SourcePath, mocksCfg.BaseDirectory, gcfg.SignaturesFilename)
+	if gcfg.SourcePath != "" {
+		mockSigsPath = path.Join(gcfg.SourcePath, gcfg.SignaturesFilename)
+	}
```

Comment:
issue: from what I understand, it shouldn't be needed at this point to still have a condition. `mocksPath` contains the value we need.

```suggestion
	mockSigsPath := path.Join(mocksPath, gcfg.SignaturesFilename)
```